### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by AntiPontifex**
See title

### CODE REVIEW

File: data/channels.csv [hunk 1]
```diff
@@ -2918,106 +2918,106 @@ BelqeesTV.ye,Belqees TV,قناة بلقيس,,Tawakkol Karman,YE,,Istanbul,c/YE,a
...
-BenTreTV.vn,Ben Tre TV,Bến Tre TV,,Ben Tre Radio and Television Station,VN,VN-50,,c/VN,vie,,FALSE,1985-07-22,,,http://thbt.vn/,https://i.postimg.cc/KYT0Czv3/Logo-THBT-B-n-Tre-m-i-2022-Logo-m-n-h-nh-B-n-1.png
+BenTreTV.vn,Ben Tre TV,Bến Tre TV,,Ben Tre Radio and Television Station,VN,VN-50,Ben Tre,c/VN,vie,general,FALSE,1985-07-22,,,http://thbt.vn/,https://i.postimg.cc/KYT0Czv3/Logo-THBT-B-n-Tre-m-i-2022-Logo-m-n-h-nh-B-n-1.png
 BenueTelevision.ng,Benue Television,,,,NG,,Makurdi,c/NG,eng,general,FALSE,,,,https://benuetv.be.gov.ng/,https://i.imgur.com/gP5aYgG.jpg
-BeratTV.tr,Berat TV,,,,TR,,,c/TR,tur,,FALSE,,,,,https://i.imgur.com/BxxH8wE.png
-BerbereTV.fr,Berbere TV,Berbère TV,,BRTV Group,FR,,Paris,c/FR;r/MAGHREB,ara;fra,culture,FALSE,,,,http://www.brtvpro.tv/,https://i.imgur.com/Uc2PAmL.png
-BerenteTV.hu,Berente TV,,,,HU,,,c/HU,hun,general,FALSE,,,,https://berente.hu,https://i.imgur.com/3H3Ct9f.png
+BeratTV.tr,Berat TV,,,,TR,,Istanbul,c/TR,tur,general,FALSE,,,,https://www.berattv.com.tr/,https://i.imgur.com/BxxH8wE.png
+BerbereTV.fr,Berbere TV,Berbère TV,,,FR,,Paris,c/FR,fra,culture,FALSE,,,,http://www.brtvpro.tv/,https://i.imgur.com/actf7dq.png
```
Comment (Alstruit) [inline data/channels.csv:2933]: Why replace the image with a lower quality?

Comment (AntiPontifex) [reply data/channels.csv:2933]: I don't remember doing that, Alstruit. Feel free to revert.

File: data/channels.csv [hunk 2]
```diff
@@ -2918,113 +2918,113 @@ BelqeesTV.ye,Belqees TV,قناة بلقيس,,Tawakkol Karman,YE,,Istanbul,c/YE,a
...
```
Comment (Alstruit) [inline data/channels.csv:2933]:
```suggestion
BerbereTV.fr,Berbere TV,Berbère TV,,,FR,,Paris,c/FR,fra,culture,FALSE,,,,http://www.brtvpro.tv/,https://i.imgur.com/Uc2PAmL.png

```

**Comment by PopeyeTheSai10r** [comment]
```
 3198   "BloombergTVAsia.hk" has an invalid subdivision "Singapore"
 3201   "BloombergTVBulgaria.bg" has an invalid subdivision "London"
 3203   "BloombergTVEMEALiveEvent.uk" has an invalid subdivision "London"
 3204   "BloombergTVEurope.uk" has an invalid subdivision "London"
 3205   "BloombergTVEuropeHD.uk" has an invalid subdivision "London"
 3198   BloombergTVAsia.hk: "subdivision" with value "Singapore" fails to match the required pattern: /^[A-Z]{2}-[A-Z0-9]{1,3}$/
 3201   BloombergTVBulgaria.bg: "subdivision" with value "London" fails to match the required pattern: /^[A-Z]{2}-[A-Z0-9]{1,3}$/
 3203   BloombergTVEMEALiveEvent.uk: "subdivision" with value "London" fails to match the required pattern: /^[A-Z]{2}-[A-Z0-9]{1,3}$/
 3204   BloombergTVEurope.uk: "subdivision" with value "London" fails to match the required pattern: /^[A-Z]{2}-[A-Z0-9]{1,3}$/
 3205   BloombergTVEuropeHD.uk: "subdivision" with value "London" fails to match the required pattern: /^[A-Z]{2}-[A-Z0-9]{1,3}$/
10 error(s)
```

**Review by PopeyeTheSai10r — CHANGES_REQUESTED** [review]
Update the affected lines

**Comment by AntiPontifex** [comment]
I tried: they keep reappearing. I mentioned earlier the problems I have been having with Visual Studio. Feel free to correct.

**Comment by ProPontifex** [comment]
Under my old account, I made the requested changes

**Comment by ProPontifex** [comment]
Would someone please approve this and move this one on. Thank you.

**Comment by PopeyeTheSai10r** [comment]
@AntiPontifex / @ProPontifex can you save the changes and try to push them again from a new branch?

**Review by PopeyeTheSai10r — APPROVED** [review]

**Comment by ProPontifex** [comment]
Sorry, I can't; old account got compromised. Still hoping you can salvage this somehow.

**Comment by PopeyeTheSai10r** [comment]
@ProPontifex take a look at https://github.com/iptv-org/database/compare/master...AntiPontifex and see if anything obvious is missing.

**Comment by ProPontifex** [comment]
Looks good: thank you :)

**Comment by PopeyeTheSai10r** [comment]
@AntiPontifex / @ProPontifex I think you should get in the habit of creating branches and making changes to your branches instead of editing the main branch directly.

**Comment by ProPontifex** [comment]
Imagine if I was able to auto-merge!

**Pull request closed without merging**